src/styles/_landing.scss

```diff
@@ -1,30 +1,45 @@
 #landing {
   #hero {
     position: relative;
-    background-image: url('/assets/img/landing/hero-2000px.jpg');
     background-size: cover;
     background-position-x: 80%;
     height: 800px;
     overflow: hidden;
     @media (max-width: 600px) {
       height: 700px;
-      &:after {
-        content: ('');
-        position: absolute;
-        width: 100%;
-        height: 100%;
-        background-image: url('/assets/img/landing/background-chair-overlay.png');
-        background-size: cover;
-        top: 20%;
-        transform: rotateX(180deg);
-        z-index: 2;
+      >div {
+        background: linear-gradient(to top, rgba(0,0,0,1) 0%,rgba(0,0,0,0) 80%);
```

Comment:
Using a PNG as gradient is always a bad idea. Use a CSS gradient for
1) avoids the need for loading yet another image
2) renders instantly as soon the CSS lands
3) better colour blend, PNG can have issues
4) browsers have to work hard to blend a transparent image with other images, a gradient is mathematically easier to process